docs/docs/sourcing-from-flotiq.md

```diff
@@ -0,0 +1,264 @@
+
+```graphql
+query {
+  allBlogPost {
+    nodes {
+      id
+      slug
+      category {
+        name
+      }
+    }
+  }
+}
+```
+
+The next step is implementing these queries in your `gatsby-node.js` file:
+
+```javascript:title=gatsby-node.js
+const path = require(`path`)
+const { createFilePath } = require(`gatsby-source-filesystem`)
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  const blogPost = path.resolve(`./src/templates/blog-post.js`)
+  const result = await graphql(`
+    query GetBlogPosts {
+      allBlogpost(sort: { fields: flotiqInternal___createdAt, order: DESC }) {
+        edges {
+          node {
+            headerImage {
+              extension
+              id
+            }
+            content
+            id
+            slug
+            title
+          }
+        }
+      }
+    }
+  `)
+
+  if (result.errors) {
+    throw result.errors
+  }
+
+  // Create blog posts pages.
+  const posts = result.data.allBlogpost.edges
+  posts.forEach((post, index) => {
+    const previous = index === posts.length - 1 ? null : posts[index + 1].node
+    const next = index === 0 ? null : posts[index - 1].node
+
+    createPage({
+      path: post.node.slug,
+      component: blogPost,
+      context: {
+        slug: post.node.slug,
+        previous,
+        next,
+      },
+    })
+  })
+}
+
+exports.onCreateNode = ({ node, actions, getNode }) => {
+  const { createNodeField } = actions
+
+  if (node.internal.type === `MarkdownRemark`) {
+    const value = createFilePath({ node, getNode })
+    createNodeField({
+      name: `slug`,
+      node,
+      value,
+    })
+  }
+}
+```
+
+Now you'll want to let Gatsby create appropriate pages for your Content Objects. This example uses a `blogPost` component as a template, which can also be found below:
+
+```javascript:title=blog-post.js
+import React from "react"
+import { Link, graphql } from "gatsby"
+
+import { rhythm, scale } from "../utils/typography"
+
+class BlogPostTemplate extends React.Component {
+  render() {
+    const post = this.props.data.blogpost
+    const siteTitle = this.props.data.site.siteMetadata.title
+    const { previous, next } = this.props.pageContext
+
+    return (
+      <article>
+        <header>
+          <h1
+            style={{
+              marginTop: rhythm(1),
+              marginBottom: 0,
+            }}
+          >
+            {post.title}
+          </h1>
+          <p
+            style={{
+              ...scale(-1 / 5),
+              display: `block`,
+              marginBottom: rhythm(1),
+            }}
+          ></p>
+        </header>
+        {post.headerImage && post.headerImage[0] && (
+          <img
+            src={`${process.env.GATSBY_FLOTIQ_BASE_URL}/image/1920x0/${post.headerImage[0].id}.${post.headerImage[0].extension}`}
+            alt="test"
```

Comment:
Is there a better alt text value you can query and pull in to show people how to do it right?